### ISSUE TITLE
WARNING: An illegal reflective access operation has occurred

### ISSUE THREAD

**Issue by geoffyuen** [TiBeN/ia-mame#14]
Even though the program works, I get this warning every time:

```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.sun.xml.bind.v2.runtime.reflect.opt.Injector (file:/usr/local/bin/ia-rcade) to method java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int)
WARNING: Please consider reporting this to the maintainers of com.sun.xml.bind.v2.runtime.reflect.opt.Injector
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

Java version:

```
openjdk version "11.0.2" 2019-01-15
OpenJDK Runtime Environment 18.9 (build 11.0.2+9)
OpenJDK 64-Bit Server VM 18.9 (build 11.0.2+9, mixed mode)
```

My system: Macbook Pro 2017, Macos Mojave

**Comment by TiBeN** [comment]
OK, this is confirmed, I have the same issue with JDK 11 on Linux. Java is in a deprecation process of some parts of its API. Pretty annoying. I'll take a look and fix this asap.

Thanks for reporting me this issue.

**Comment by TiBeN** [comment]
Fixed by da82ad4
